karma-edit-story.config.cjs

```diff
@@ -143,6 +143,8 @@ module.exports = function (config) {
       jasmine: {
         timeoutInterval: 10000,
       },
+      useIframe: false,
+      runInParent: true,
```

Comment:
Could you elaborate on the reason for this change?

Looks reasonable as it seems to fix some issues with tooltip positioning, but curious to know how this relates to this PR.

---

Adding the [docs](http://karma-runner.github.io/5.2/config/configuration-file.html) for reference:

> useIframe (default: `true`)
> If true, Karma runs the tests inside an iFrame. If false, Karma runs the tests in a new window. Some tests may not run in an iFrame and may need a new window to run.

> runInParent (default: `false`)
> If true, Karma runs the tests inside the original window without using iframe. It will load the test scripts dynamically.

Reply:
I see now that if `runInParent` is `true`, `customClientContextFile	` needs to be used instead of `customContextFile`.

So it sounds like using `customContextFile` did not work?

Reply:
`customContextFile` didn't work. I couldn't work out why. Using `runInParent` and `useIframe` forces the use of `customClientContextFile`